client/src/containers/requester/role/MemberList.css

```diff
@@ -12,3 +12,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ----------------------------------------------------------------------------- */
+
+#owner-popup-box,#member-popup-box {
+    border-radius: 8px;
```

Comment:
Nitpick: 2 space indent and all files should end with a newline. If you hover over the red icon below it indicates that the file has no newline at the end.